# -*- coding: utf-8 -*-
# Generated by Django 1.10.6 on 2017-04-24 20:28
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('registration', '0002_auto_20170420_1825'),
    ]

    operations = [
        migrations.AlterField(
            model_name='report',
            name='files',
            field=models.ManyToManyField(null=True, to='registration.UserFiles'),
        ),
        migrations.AlterField(
            model_name='siteuser',
            name='user_type',
            field=models.CharField(choices=[('CMP_USR', 'Company'), ('INV_USR', 'Investor')], default='INV_USR', max_length=7),
        ),
        migrations.AlterField(
            model_name='siteuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
